LICENSE-3rdparty.csv

```diff
@@ -1,1408 +1,1408 @@
+core,github.com/josharian/native,MIT,Josh Bleecher Snyder
+core,github.com/json-iterator/go,MIT,json-iterator
+core,github.com/jstemmer/go-junit-report,MIT,Joel Stemmer
+core,github.com/jstemmer/go-junit-report/formatter,MIT,Joel Stemmer
+core,github.com/jstemmer/go-junit-report/parser,MIT,Joel Stemmer
+core,github.com/kardianos/osext,BSD-3-Clause,The Go Authors
+core,github.com/kjk/lzma,BSD-3-Clause,Andrei Vieru
+core,github.com/klauspost/compress/flate,BSD-3-Clause,Klaus Post | The Go Authors
+core,github.com/klauspost/compress/fse,BSD-3-Clause,Klaus Post | The Go Authors
+core,github.com/klauspost/compress/gzip,BSD-3-Clause,Klaus Post | The Go Authors
+core,github.com/klauspost/compress/huff0,BSD-3-Clause,Klaus Post | The Go Authors
+core,github.com/klauspost/compress/snappy,BSD-3-Clause,The Snappy-Go Authors
+core,github.com/klauspost/compress/zstd,BSD-3-Clause,Caleb Spare | Klaus Post | The Go Authors
+core,github.com/klauspost/compress/zstd/internal/xxhash,MIT,Caleb Spare | Klaus Post | The Go Authors
+core,github.com/klauspost/pgzip,MIT,Klaus Post
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver/endpoints/handlers,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver/installer,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver/registry/rest,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/cmd,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/cmd/server,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/dynamicmapper,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/provider,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/registry/custom_metrics,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/registry/external_metrics,Apache-2.0,UNKNOWN
+core,github.com/lxn/walk,BSD-3-Clause,The walk authors
+core,github.com/lxn/win,BSD-3-Clause,The win authors
+core,github.com/magiconair/properties,BSD-2-Clause,Frank Schroeder
+core,github.com/mailru/easyjson,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/bootstrap,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/buffer,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/easyjson,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/gen,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/jlexer,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/jwriter,MIT,Mail.Ru Group
+core,github.com/mailru/easyjson/parser,MIT,Mail.Ru Group
+core,github.com/mattn/go-colorable,MIT,Yasuhiro Matsumoto
+core,github.com/mattn/go-isatty,MIT,Yasuhiro MATSUMOTO <mattn.jp@gmail.com>
+core,github.com/mattn/go-runewidth,MIT,Yasuhiro Matsumoto
+core,github.com/matttproud/golang_protobuf_extensions/pbutil,Apache-2.0,UNKNOWN
```

Comment:
Matt T. Proud (matt.proud@gmail.com)

Reply:
I added "NOTICE" to the list of files too..